value.go

```diff
@@ -26,6 +26,9 @@ type value struct {
 	Descriptor protoreflect.FieldDescriptor
 	// Constraints are the individual evaluators applied to a value
 	Constraints evaluators
+	// NestedConstraints are constraints applied to constraints nested under a
```

Comment:
```suggestion
	// NestedConstraints are constraints applied to messages nested under a
```

Not sure what you're trying to say here exactly.

Reply:
The only reason why `NestedConstraints` exists is to make it possible to apply the field-level constraints for a field without applying all of the constraints in nested submessages. Anyway, updated the comment.